README.md

```diff
@@ -1,2 +1,231 @@
+| `envRenderSecret`                         | Sensible environment variables passed to pods and stored as secret | `{}`                               |
+| `extraSecretMounts`                       | Secret mounts to get secrets from files instead of env vars | `[]`                                      |
+| `extraVolumeMounts`                       | Additional volume mounts                      | `[]`                                                    |
+| `configuration.pythonFileName`            | Path of the python file to run                | `basic.py`                                              |
+| `configuration.processesCount`            | Number of concurrent processes to run         | `1`                                                     |
+| `configuration.workersPerProcess`         | Number of workers per process                 | `1`                                                     |
+| `configuration.keepAlive`                 | Keep the container process alive after dataflow executing ended to prevent a container restart by Kubernetes | `true` |
+| `configuration.configMap.create`          | Create a configmap to store python file(s)    | `true`                                                  |
+| `configuration.configMap.customName`      | Configmap which has python file(s) created manually | ``                                                |
+| `configuration.configMap.files.pattern`   | Files to store in the ConfigMap to be created | `examples/*`                                            |
+| `configuration.configMap.files.tarName`   | Tar file to store in the ConfigMap to be created | ``                                                   |
+
+
+### Example running basic.py obtained from a Configmap created by Helm
+
+```yaml
+configuration:
+  pythonFileName: "basic.py"
+  configMap:
+    files:
+      pattern: "examples/*"
+      tarName:
+```
+
+### Example running pagerank.py obtained from a Configmap created by Helm
+
+```yaml
+configuration:
+  pythonFileName: "examples/pagerank.py"
+  configMap:
+    files:
+      pattern: 
+      tarName: "examples.tar"
+```
+
+In this example, we store a tar file in the configmap. This is useful when your python script needs a tree of nested files and directories. 
+
+Following our example, the tar file has this content:
+```console
+├── examples
+│   ├── anomaly_detector.py
+│   ├── basic.py
+│   ├── events_to_parquet.py
+│   ├── pagerank.py
+│   ├── sample_data
+│   │   ├── easy_count.txt
+│   │   ├── graph.txt
+│   │   ├── lyrics.txt
+│   │   └── wordcount.txt
+│   ├── search_session.py
+│   ├── sleepyworkers.py
+│   ├── subflow.py
+│   ├── translator.py
+│   ├── twitter_stream.py
+│   ├── utils
+│   │   ├── fake_events.py
+│   │   ├── __init__.py
+│   │   └── twitter.py
+│   ├── wikistream.py
+│   └── wordcount.py
+```
+So, because that tar file is going to be extracted to the container working directory then the container is going to have that tree available to work with.
```

Comment:
```suggestion
Since that tar file is going to be extracted to the container working directory then the container is going to have that directory tree available to work with.
```